### DISCUSSION TITLE
Import collection.json from command line interface

### DISCUSSION THREAD

**Discussion by tobyworks** [pocketbase/pocketbase#3046] [Q&A] [answered]
I'm working on a project that will be dockerized and as backend for the project im using pocketbase. Is there a way to import collections.json file right from the command like it is possible to create admin account from the command line?

ideally it would be something like this:

```
RUN wget https://github.com/pocketbase/pocketbase/releases/download/v0.17.1/pocketbase_0.17.1_linux_arm64.zip \
    && unzip pocketbase_0.17.1_linux_arm64.zip \
    && chmod +x /app/pocketbase \
    && /app/pocketbase migrate \
    && /app/pocketbase admin create admin@myproject.local supersecretpassword
    && /app/pocketbase bootstrap collections.json ## <-- does something like this exist???
```

**Comment by ganigeorgiev** [comment, marked as answer]
>  Is there a way to import collections.json file right from the command

No (at least not out of the box), but for such purposes I recommend using Migrations (either in [Go](https://pocketbase.io/docs/go-migrations/) and/or [JS](https://pocketbase.io/docs/js-migrations/)).

If you are using the prebuilt executable, then the JS migrations are used and they are stored in `pb_migrations` directory. This directory is safe to be committed to version control and/or included in your image.

You can generate a migration with snapshot of your current collections configuration (basically the same as the exported json from the Admin UI) by running `./pocketbase migrate collections` (for more details see [Collections snapshot](https://pocketbase.io/docs/js-migrations/#collections-snapshot)).

Migrations are automatically loaded and executed on `serve`.
